src/index.js

```diff
@@ -770,7 +770,7 @@ class NearProvider {
                 });
             return '0x' + result;
         } catch (e) {
-            return ;
+            return e;
```

Comment:
should this `raise e;` instead? or the API requires returning errors via return value?

Reply:
I believe it requires returning it, which is why all the methods return the error, but I'll double check and fix if otherwise.